SpatialGDK/Extras/core-sdk.version

```diff
@@ -1 +1 @@
-13.3.1
+13.5.0
```

Comment:
Is this necessary for locator v2 to work? I'll defer to the GDK team but I'd be hesitant to upgrade core SDK versions without more testing.

Reply:
Yes this is. Locator v2 doesn't exist in previous versions of the SDK